internal/app/machined/internal/phase/phase.go

```diff
@@ -128,7 +128,7 @@ func (r *Runner) runTask(task Task, errCh chan<- error) {
 
 	defer func() {
 		if r := recover(); r != nil {
-			buf := make([]byte, 8192)
+			buf := make([]byte, 512) // using small buffer here, as kmsg has its limits
```

Comment:
kmsg has limit slightly less than 1 KB: https://github.com/torvalds/linux/blob/master/kernel/printk/printk.c#L450

Reply:
this fix is in fact good on its own: it was preventing panic messages to be printed in any phase is traceback was long enough